### PR TITLE
Update blogpost: How can i determine if a folder exists on my computer

### PR DESCRIPTION
**Pull request by termdew**
# PR Summary
This PR contains an update for the article named "How can i determine if a folder exists on my computer?"

## PR Checklist

- Do you want this post to be published on a specific date?: Y/N [Specify date]: N
- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] Includes content related an open issue
  - see [Closing issues using keywords][key].
- [ ] This PR is ready to merge and is not **Work in Progress**

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+post_title: Determine if a folder exists
+username: tfl@psp.co.uk
+Catagories: PowerShell
+tags: File, Test-Path
+Summary: How can I determine if a folder exists?
+---
+
+**Q:** Is there any way to determine whether or not a specific folder exists on a computer?
+**A:**  There are loads of ways you can do this.
+A simple way is to use the power of array handling in PowerShell.
```
Comment (doctordns) [inline Posts/2021/03/ScriptingGuy.0007.folderexists.md:11]: ??

Comment (doctordns) [reply Posts/2021/03/ScriptingGuy.0007.folderexists.md:11]: The easiest way to to this is to use the `Test-Path` cmdlet.

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+```powershell
+$Folder = 'C:\Windows'
+"Test to see if folder [$Folder]  exists"
+if (Test-Path -Path $Folder) {
+    "Path exists!"
+} else {
+    "Path doesn't exist."
+}
+```
+This is similar to the `-d` parameter in Bash.
```
Comment (doctordns) [inline Posts/2021/03/ScriptingGuy.0007.folderexists.md:28]: which command does -D refer to?

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## For More Information
+
+And for more information on `Test-Path` see the [Test-Path](https://docs.microsoft.com/powershell/module/microsoft.powershell.management/test-path) help page.
+
+## Summary
+
+So as you saw, `Test-Path` tests the existence of a path and returns a boolean value.
+This return value can be evaluated in a if-statement for example.
```
Comment (doctordns) [inline Posts/2021/03/ScriptingGuy.0007.folderexists.md:37]: IF statement (remove the dash)

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+$Folder = 'C:\Windows'
+"Test to see if folder [$Folder]  exists"
+if (Test-Path -Path $Folder) {
+    "Path exists!"
+} else {
+    "Path doesn't exist."
+}
+```
+This is similar to the `-d` parameter in Bash.
+
```
Comment (doctordns) [inline Posts/2021/03/ScriptingGuy.0007.folderexists.md:29]: Show the output not just the commands.

**Review by doctordns — CHANGES_REQUESTED** [review]
Good start and thanks. 

Some tidying up to do but close to being ready.

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+post_title: Determine if a folder exists
+username: tfl@psp.co.uk
```
Comment (sdwheeler) [inline Posts/2021/03/ScriptingGuy.0007.folderexists.md:3]: This should be your username not Thomas's.

Comment (doctordns) [reply Posts/2021/03/ScriptingGuy.0007.folderexists.md:3]: LOL-he is free to give me credit as I did write much of the post! :-)

**Comment by doctordns** [comment]
@termdew if you can address the minor issues raised, we can get this PR approved and the blog post posted.
Ping me privately if you have any problems.

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md (hunk 1 above)
Comment (termdew) [reply Posts/2021/03/ScriptingGuy.0007.folderexists.md]: resolved.

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md (hunk 2 above)
Comment (termdew) [reply Posts/2021/03/ScriptingGuy.0007.folderexists.md]: It's used in Bash IF statements. Resolved.

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md (hunk 3 above)
Comment (termdew) [reply Posts/2021/03/ScriptingGuy.0007.folderexists.md]: Resolved.

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md (hunk 4 above)
Comment (termdew) [reply Posts/2021/03/ScriptingGuy.0007.folderexists.md:28]: It's difficult to show "true". Resolved.

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md (hunk 5 above)
Comment (termdew) [reply Posts/2021/03/ScriptingGuy.0007.folderexists.md]: Resolved ;-)

**Comment by doctordns** [comment]
looks good to me

File: Posts/2021/03/ScriptingGuy.0007.folderexists.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+---
+post_title: Determine if a folder exists
+username: fabian.baumanis@gmx.de
```
Comment (sdwheeler) [inline Posts/2021/03/ScriptingGuy.0007.folderexists.md:3]: @termdew Before we can merge this, you need to create your WordPress account. I don't see an account for you in WordPress. For instructions, see https://github.com/PowerShell/Community-Blog/wiki/Becoming-a-contributor

**Review by sdwheeler — CHANGES_REQUESTED** [review]

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**